components/UserPermissions.php

```diff
@@ -82,17 +82,17 @@ public static function canEditUser(User $user)
         if (\Yii::$app->user->isGuest) {
             return false;
         }
-
-        $currentUserID = \Yii::$app->user->getId();
-
-        if ((int)$user->id === $currentUserID) {
```

Comment:
It was an optimization not to invoke RBAC in this case.

Reply:
Fixed.